controllers/slicegateway/reconciler.go

```diff
@@ -245,13 +245,17 @@ func (r *SliceGwReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		if err != nil {
 			return ctrl.Result{}, err
 		}
+		secretVersion := 1
```

Comment:
nit: why is this variable needed?
directly passing `vpnKeyRotation.Spec.RotationCount` to won't work ?

Reply:
This is needed because what in gateway testcases we are not creating vpnkeyrotation so it is fine to initialise it with one in such cases.